fedimintd/src/fedimintd.rs

```diff
@@ -188,8 +189,13 @@ impl Fedimintd {
         let s = self
             .with_module(LightningInit)
             .with_module(MintInit)
-            .with_module(WalletInit)
-            .with_module(MetaInit);
+            .with_module(WalletInit);
+
+        let s = if !is_env_var_set(FM_DISABLE_META_MODULE_ENV) {
+            s.with_module(MetaInit)
+        } else {
+            s
+        };
```

Comment:
Not great, but ok for now. Maybe our tests should just specify which modules they want during setup?

Reply:
The end goal, IMO, is DKG/tests etc. specifying what module instances they want, without anything being enabled "by default" (other than maybe in the DKG UI level).

But even then we are going to need conditional logic in some compatibility tests to not enable certain modules (or skip test altogether if it requires it), if previous version simply can't handle given module. It will however remove a need for disabling some modules via env vars, and make it as an explicit step.